### PR TITLE
Split tests into separate files & other cleanup

### PR DESCRIPTION
**Pull request by deiwin**

### CODE REVIEW

**Comment by urmastalimaa** [comment]
+1

File: pull_request_event_test.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+				"X-Github-Event": "pull_request",
+			}
+		})
+
+		Context("with the PR being closed", func() {
+			requestJSON.Is(func() string {
+				return PullRequestsEvent("closed")
+			})
+
+			It("succeeds with a message that says the request is ignored as PR not opened/synchronized", func() {
```
Comment (willmore) [inline pull_request_event_test.go]: Rephrase: "succeeds with 'ignored' response"

File: review_command_test.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			Context(desc, func() {
+				requestJSON.Is(func() string {
+					return IssueCommentEvent(comment)
+				})
+
+				itMarksCommitPeerReviewed()
+			})
+		}
+
+		Context("with +1 as a part of a number", func() {
```
Comment (willmore) [inline review_command_test.go]: Nice cases :)

File: authentication_test.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+			requestJSON.Is(func() string {
+				return "{}"
+			})
+			headers.Is(func() map[string]string {
+				return map[string]string{
+					"X-Hub-Signature": validSignature,
+				}
+			})
+
+			It("succeeds with a message that says the request is ignored", func() {
```
Comment (willmore) [inline authentication_test.go]: "succeeds with 'ignore request' response"
here and line 81.

**Comment by willmore** [comment]
Other than comments on a couple of verbose statements, +1

File: authentication_test.go (hunk 3 above)
Comment (deiwin) [inline authentication_test.go]: https://github.com/salemove/github-review-helper/pull/16/commits/ac25f4f2ce146525c17da4f2a7b04795a71a680a

File: pull_request_event_test.go (hunk 1 above)
Comment (deiwin) [inline pull_request_event_test.go]: https://github.com/salemove/github-review-helper/pull/16/commits/ac25f4f2ce146525c17da4f2a7b04795a71a680a

**Comment by deiwin** [comment]
!squash

**Pull request merged**